From bd701d5ccfaa84dc86dfccbf56212fa47fa0578f Mon Sep 17 00:00:00 2001
From: Daniel Templeton <templedf@apache.org>
Date: Fri, 2 Dec 2016 13:48:18 -0800
Subject: [PATCH 2102/2748] CLOUDERA-BUILD. Excluding flaky test
 TestAMRMClient.testAllocation

This test should be fixed and reinstated. Excluding for CDH 5.10.

Change-Id: Icc11caac578fd1c315e06f1b1385fae817156ac6
---
 .../yarn/client/api/impl/TestAMRMClient.java       |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/src/test/java/org/apache/hadoop/yarn/client/api/impl/TestAMRMClient.java b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/src/test/java/org/apache/hadoop/yarn/client/api/impl/TestAMRMClient.java
index e0137f7..1515074 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/src/test/java/org/apache/hadoop/yarn/client/api/impl/TestAMRMClient.java
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/src/test/java/org/apache/hadoop/yarn/client/api/impl/TestAMRMClient.java
@@ -90,6 +90,7 @@
 import org.junit.AfterClass;
 import org.junit.Before;
 import org.junit.BeforeClass;
+import org.junit.Ignore;
 import org.junit.Test;
 import org.mockito.invocation.InvocationOnMock;
 import org.mockito.stubbing.Answer;
@@ -509,6 +510,7 @@ public void testAMRMClientMatchStorage() throws YarnException, IOException {
     }
   }
   
+  @Ignore
   @Test (timeout=60000)
   public void testAllocationWithBlacklist() throws YarnException, IOException {
     AMRMClientImpl<ContainerRequest> amClient = null;
-- 
1.7.9.5

